data/state/proto/peerAccountData.proto

```diff
@@ -21,22 +21,22 @@ message TimePeriod {
 
 // SignRate is used to keep the number of success and failed signings
 message SignRate {
-    uint32 NrSuccess = 1;
-    uint32 NrFailure = 2;
+    uint32 NumSuccess = 1;
+    uint32 NumFailure = 2;
 }
 
 // ValidatorApiResponse represents the data which is fetched from each validator for returning it in API call
 message ValidatorApiResponse {
-	uint32 NrLeaderSuccess = 1 [(gogoproto.jsontag) = "nrLeaderSuccess"];
-	uint32 NrLeaderFailure = 2 [(gogoproto.jsontag) = "nrLeaderFailure"];
-	uint32 NrValidatorSuccess = 3 [(gogoproto.jsontag) = "nrValidatorSuccess"];
-	uint32 NrValidatorFailure = 4 [(gogoproto.jsontag) = "nrValidatorFailure"];
-	float Rating = 5 [(gogoproto.jsontag) = "rating"];
-	float TempRating = 6 [(gogoproto.jsontag) = "tempRating"];
-    uint32 TotalNrLeaderSuccess = 7 [(gogoproto.jsontag) = "totalNrLeaderSuccess"];
-    uint32 TotalNrLeaderFailure = 8 [(gogoproto.jsontag) = "totalNrLeaderFailure"];
-    uint32 TotalNrValidatorSuccess = 9 [(gogoproto.jsontag) = "totalNrValidatorSuccess"];
-    uint32 TotalNrValidatorFailure = 10 [(gogoproto.jsontag) = "totalNrValidatorFailure"];
+    float TempRating = 1 [(gogoproto.jsontag) = "tempRating"];
+	uint32 NumLeaderSuccess = 2 [(gogoproto.jsontag) = "NumLeaderSuccess"];
```

Comment:
maybe ident this? (github does not render properly. Suggestion to use spaces?)

Reply:
fixed

Reply:
not fixed, please check https://github.com/ElrondNetwork/elrond-go/blob/0639acafd38ec42c74a604c16e0fef479dde04e5/data/state/proto/peerAccountData.proto